lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_get_udids.rb

```diff
@@ -0,0 +1,93 @@
+
+        app_id = params[:app]
+        udids = fad_api_client.get_udids(app_id)
+        write_udids_to_file(udids, params[:output_file])
+        UI.success("🎉 App Distribution tester UDIDs written to: #{params[:output_file]}")
+      end
+
+      def self.write_udids_to_file(udids, output_file)
+        File.open(output_file, 'w') do |f|
+          f.write("Device ID\tDevice Name\tDevice Platform\n")
+          udids.each do |tester_udid|
+            f.write("#{tester_udid[:udid]}\t#{tester_udid[:name]}\t#{tester_udid[:platform]}\n")
+          end
+        end
+      end
+
+      def self.description
+        "Download the UDIDs of your Firebase App Distribution testers"
+      end
+
+      def self.authors
+        ["Lee Kellogg"]
+      end
+
+      # supports markdown.
+      def self.details
+        "Export your testers' device names and identifiers in a CSV file, so you can add them your app's provisioning profile. This file can be imported into your Apple developer account using the Register Multiple Devices option. See the [App Distribution docs](https://firebase.google.com/docs/app-distribution/ios/distribute-console#register-tester-devices) for more info."
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :app,
+                                       env_name: "FIREBASEAPPDISTRO_APP",
+                                       description: "Your app's Firebase App ID. You can find the App ID in the Firebase console, on the General Settings page",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :output_file,
+                                       env_name: "FIREBASEAPPDISTRO_OUTPUT_FILE",
+                                       description: "The path to the file where the tester UDIDs will be written",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :firebase_cli_token,
+                                       description: "Auth token for firebase cli",
+                                       optional: true,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :service_credentials_file,
+                                       description: "Path to Google service account json",
+                                       optional: true,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :debug,
+                                       description: "Print verbose debug output",
+                                       optional: true,
+                                       default_value: false,
+                                       is_string: false)
+        ]
+      end
+
+      def self.is_supported?(platform)
```

Comment:
Where is this called? And where do you get platform from?

Reply:
It actually still works if platform is set to Android, you just get this message:

```
[12:28:41]: Action 'firebase_app_distribution_get_udids' isn't known to support operating system 'android'.
```

Reply:
Nice thanks for the explanation 🙂